images/autobump/pr-creator/main.go

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Do we need to copy the code here?

Maybe we can just use the go library `k8s.io/test-infra/robots/pr-creator`, WDYT?

Reply:
For the pr-creator we needed to create a new custom image for the commit signing. We needed to include the `pr-creator` binary in that image since the `autobump.sh` calls it. We can totally call in the library of `k8s.io/test-infra/robots/pr-creator/updater` though.